command/acl/token/update/token_update_test.go

```diff
@@ -219,27 +228,13 @@ func TestTokenUpdateCommandWithAppend(t *testing.T) {
 	)
 	require.NoError(t, err)
 
-	// create a token
-	token, _, err := client.ACL().TokenCreate(
-		&api.ACLToken{Description: "test", Policies: []*api.ACLTokenPolicyLink{{Name: policy.Name}}},
-		&api.WriteOptions{Token: "root"},
-	)
-	require.NoError(t, err)
-
 	//secondary policy
 	secondPolicy, _, policyErr := client.ACL().PolicyCreate(
 		&api.ACLPolicy{Name: "secondary-policy"},
 		&api.WriteOptions{Token: "root"},
 	)
 	require.NoError(t, policyErr)
```

Comment:
Can this be removed as well?

Reply:
this second policy is used by some tests